docker/incremental_load.sh.tpl

```diff
@@ -237,3 +237,7 @@ function read_variables() {
 # List of 'tag_layer' statements for all tags.
 # This generated and injected by docker_*.
 %{tag_statements}
+
+# An optional "docker run" statement for invoking a loaded container.
+# This generated and injected by docker_*.
+%{run_statements}
```

Comment:
Maybe make the run optional via an argument of the runner instead?

So I can activate the new behavior with `bazel run //my:target -- --run` and there is no need for the legacy_XXX attr in the rule.